lib/pure/times.nim

```diff
@@ -16,7 +16,7 @@
                       # of the standard library!
 
 import
-  strutils
+  strutils, pegs
```

Comment:
OK - didn't realize it would affect compile times. Just using strutils now. I also fixed a bug in doing so, which was helpful.